src/MovieContainer/MovieContainer.js

```diff
@@ -1,20 +1,37 @@
 import React from 'react';
 import MovieCard from '../MovieCard/MovieCard';
+import UserMovieCard from '../UserMovieCard/UserMovieCard';
 import './MovieContainer.css';
 
-const MovieContainer = ({movies}) => {
-	console.log(movies)
-	const movieCards = movies.map((movie, i) => {
-		return (
-			<MovieCard
-				title={movie.title}
-				averageRating={movie.average_rating}
-				poster={movie.poster_path}
-				id={movie.id}
-				key={i}
-			/>
-		)
-	})
+const MovieContainer = ({appState, usersRatings}) => {
+	let movieCards;
+
+	if (appState.userID) {
```

Comment:
Maybe we could switch this to a ternary inside of our return statement so there would be an optional prop for userRatings